clang/Type.d

```diff
@@ -130,13 +169,10 @@ struct Type
         return ArrayType(this);
     }
 
-    @property bool isArray ()
+    @property string toString() const
     {
-        return
-            kind == CXTypeKind.CXType_ConstantArray ||
-            kind == CXTypeKind.CXType_IncompleteArray ||
-            kind == CXTypeKind.CXType_VariableArray ||
-            kind == CXTypeKind.CXType_DependentSizedArray;
+        import std.format: format;
+        return format("Type(kind = %s, spelling = %s, isConst = %s)", kind, spelling, isConst);
     }
 }
 
```

Comment:
That's understandable, but with the amount of code in your pull requests, every diff counts. It's better to make as few changes as possible, especially when it comes to style related changes. I know that can be hard but it's better to leave that for a completely separate pull request. It's also in your best interest because it will make my life easier and speed up the review process 😃.